src/User/User.ts

```diff
@@ -0,0 +1,32 @@
+import firebase from "firebase";
+
+type getToken =
+  | ((forceRefresh?: boolean | undefined) => Promise<string>)
+  | undefined;
+
+type IUSER = firebase.User | null;
+interface Profile {
+  email: string | undefined | null;
```

Comment:
We are only supporting `auth` based on email and password. Can we do without `undefined` and `null` here?

Reply:
https://github.com/pesto-students/juntos-frontend/pull/18#discussion_r649158858